cli/src/cli_util.rs

```diff
@@ -2293,7 +2293,12 @@ See https://jj-vcs.github.io/jj/latest/working-copy/#stale-working-copy \
             .commits(repo.store())
             .try_collect()?;
 
-        if !root_conflict_commits.is_empty() {
+        let should_show_resolve_conflicts_hint = self
+            .settings()
+            .get_bool("hints.howto-resolve-conflicts")
+            .unwrap_or(true);
```

Comment:
nit: can you add the default value to `config/*.toml` instead of ignoring error? Suppose the list will grow, it might make sense to add `config/hints.toml`.

Reply:
That's done - I've gone with `hints.toml`